en/configure-tidb-binlog-drainer.md

```diff
@@ -24,7 +24,7 @@ The following table contains all configuration parameters available for the `tid
 | `storageClassName` | `storageClass` used by the drainer. `storageClassName` refers to a type of storage provided by the Kubernetes cluster, which might map to a level of service quality, a backup policy, or to any policy determined by the cluster administrator. Detailed reference: [storage-classes](https://kubernetes.io/docs/concepts/storage/storage-classes) | `local-storage` |
 | `storage` | The storage limit of the drainer Pod. Note that you should set a larger size if `db-type` is set to `pb` | `10Gi` |
 | `disableDetect` |  Determines whether to disable casualty detection | `false` |
-| `initialCommitTs` |  Used to initialize a checkpoint if the drainer does not have one | `0` |
+| `initialCommitTs` |  Used to initialize a checkpoint if the drainer does not have one. The value is a string type, such as `"424364429251444742"` | `"-1"` |
```

Comment:
@csuzhangxc Please also help update the `deploy-tidb-binlog.md` which is the doc that users refer to directly.

Reply:
did you mean to update the example in `deploy-tidb-binlog.md`, but this param is critical and if it may cause data lost if the user coyp & paste it.

or add a NOTE for this param?

Reply:
updated `deploy-tidb-binlog.md` in https://github.com/pingcap/docs-tidb-operator/pull/1165/commits/b8c0497fdf1565bf8d74891c001fbe9306ad7fc3.